### PR TITLE
RHPAM-2101: BC Monitoring fails to start a new process in an unmanaged immutable kieserver

### PR DESCRIPTION
**Pull request by pefernan**
Adding missing config for forms engine on monitoring webapp. Additionaly removed duplicated gwt module on BC.

@tomasdavidorg @cristianonicolai @sutaakar could you please take a look?

### CODE REVIEW

**Comment by pefernan** [comment]
jenkins execute full downstream build please

**Comment by pefernan** [comment]
jenkins execute full downstream build please

File: business-central-parent/business-central-webapp/src/main/resources/org/kie/bc/KIEWebapp.gwt.xml [hunk 1]
```diff
@@ -135,9 +135,6 @@
 
   <inherits name="com.google.gwt.xml.XML"/>
 
-  <!-- Form modeler.-->
-  <inherits name="org.kie.workbench.common.forms.dynamic.DynamicFormsClient"/>
```
Comment (sutaakar) [inline business-central-parent/business-central-webapp/src/main/resources/org/kie/bc/KIEWebapp.gwt.xml]: Why is this piece deleted (I am not much familiar with GWT)?

**Comment by sutaakar** [comment]
Just checked that manually on OpenShift and it seems to fix the issue

**Comment by pefernan** [comment]
Jenkins execute full downstream build

**Comment by pefernan** [comment]
jenkins execute full downstream build please

**Pull request merged**